CMakeLists.txt

```diff
@@ -146,6 +146,8 @@ endif()
 option(BUILD_MISC_TESTS "Build additional tests" OFF)
 option(BUILD_FUN_UTILS "Build additional just for fun utilities" OFF)
 
+option(UNITTEST "Enable unit tests (disable if you don't have a working gmock or gtest)" ON)
```

Comment:
Add it to the INSTALL.md table of CMake options. That one is alphabetically sorted.

Reply:
Done.